lxd/identity/util.go

```diff
@@ -24,6 +24,8 @@ func AuthenticationMethodFromIdentityType(identityType string) (string, error) {
 	switch identityType {
 	case api.IdentityTypeCertificateClientRestricted, api.IdentityTypeCertificateClientUnrestricted, api.IdentityTypeCertificateServer, api.IdentityTypeCertificateMetrics:
 		return api.AuthenticationMethodTLS, nil
+	case api.IdentityTypeOIDCClient:
```

Comment:
is this actually related or dependent on the openfga drive?

Reply:
I'll double check this one.